src/Workleap.DomainEventPropagation.Tests/Workleap.DomainEventPropagation.Tests.csproj

```diff
@@ -8,11 +13,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Workleap.DomainEventPropagation.Publishing\Workleap.DomainEventPropagation.Publishing.csproj" />
     <ProjectReference Include="..\Workleap.DomainEventPropagation\Workleap.DomainEventPropagation.csproj" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
+    <PackageReference Include="Moq" Version="4.18.4" />
```

Comment:
We can later change this to FakeItEasy, but for now it's all good

Reply:
Correct, this was less trivial to address right now compared to the other changes ✌️